shared/map/TileOwnerType.js

```diff
@@ -0,0 +1,36 @@
+import { TYPES } from './tile'
+import { isPartOfEstate } from '../parcel'
+
+export class TileOwnerType {
+  constructor(owner) {
+    this.owner = owner
+  }
+
+  get(tile) {
+    const isOwner = tile.owner === this.owner
+    const hasAccess = tile.approvals.includes(this.owner)
```

Comment:
I thought about it, but the problem is that the nomenclature for this things is a bit all over the place at the moment.

We have:
- `authorizations` in the **frontend** which refer to allowances and approvals (in the Settings page)
- `/authorizations` in the **backend** which refer to an endpoint that summarizes what you can do with a particular asset
- `Approvals` in the **backend** which refer to address-to-address approvals (`UpdateManager`, `ApprovalForAll`)
- `roles` in **/shared** which refer to the role you have for a given asset (operator, approval for all, update operator, etc)
- `<Permission />` in the **frontend** which tells you if you have permission to do a certain action to an asset
- `Has access` on the map tile popup in the **frontend**. When you hover over a tile where you "have access" but you're not the owner

This last one is why I chose that name, I also didn't want to use `authorization` again as it can be confusing.

----

I think I'll do the following:
1. Change this to `isAuthorized`
2. Change the frontend popup to say `You are authorized`
3. 
**Another PR**

Reply:
Ok, I see.  I think that `isAuthorized` fits better because `access` can be thought as you can enter like physically, but maybe it is just me. 

I will go with: 
```
Change hasAccess to isAuthorized
Change the frontend popup to say You are authorized
Change all other access references
```

But, to be honest I will do it in another PR if you agree. We can create an issue and rename it later.

One more thing that you mentioned and may needs treatment could be `Approvals`. If we go with `isAuthorized` it should be renamed as `Authorizations`